### PR TITLE
#164068462 Improve test coverage

### PR DESCRIPTION
**Pull request by bevkololi**
#### What does this PR do?
Add tests in codebase to improve test coverage.

#### Description of Task to be completed?
There are some uncovered lines in the codebase, this PR adds tests to ensure that most are covered to improve the overall test coverage.

#### How should this be manually tested?
* Clone the branch and follow the following steps to [setup](https://github.com/andela/mrm_api).
* Run tests using the command `make test test="coverage report"`

#### Any background context you want to provide?
N/A 

#### Screenshots
N/A

#### What are the relevant pivotal tracker stories?
[#164068462](https://www.pivotaltracker.com/story/show/164068462)

### CODE REVIEW

File: tests/test_location/test_create_location.py [hunk 1]
```diff
@@ -16,3 +19,30 @@ def test_location_creation(self):
         """
         CommonTestCases.admin_token_assert_equal(
             self, create_location_query, create_location_response)
+
+    def test_location_creation_with_invalid_image_url(self):
+        """
+        Testing for location creation with invalid url
```
Comment (Bonifase) [inline tests/test_location/test_create_location.py]: Should be invalid image url

File: tests/test_response/test_user_response.py [hunk 2]
```diff
@@ -207,3 +209,23 @@ def test_filter_response_by_invalid_room_id(self):
...
+        """
+        Testing for when user rates without "rate" args
+        """
+        CommonTestCases.user_token_assert_in(
+            self,
+            create_rate_query_without_rate_args,
+            "Provide a rating response"
+        )
+
+    def test_text_mutation_without_text_args(self):
```
Comment (Bonifase) [inline tests/test_response/test_user_response.py]: This test function name is not clear, kindly make it descriptive

File: tests/test_room_resource/test_create_room_resource.py [hunk 3]
```diff
@@ -39,3 +39,13 @@ def test_room_resource_creation_room_id_error(self):
             resource_mutation_0_room_id,
             "Room not found"
         )
+
+    def test_create_resource_in_other_location(self):
```
Comment (Bonifase) [inline tests/test_room_resource/test_create_room_resource.py]: Make this test function name more descriptive. The name is different from what is being tested

File: tests/test_room_resource/test_delete_resource.py [hunk 4]
```diff
@@ -32,3 +32,10 @@ def test_non_existant_deleteresource_mutation(self):
             delete_non_existant_resource,
             "Resource not found"
         )
+
+    def test_deleteresource_mutation_admin_different_location(self):
```
Comment (Bonifase) [inline tests/test_room_resource/test_delete_resource.py]: Make this test function name more descriptive. The name is different from what is being tested

File: tests/test_user/test_query_user.py [hunk 5]
```diff
@@ -84,3 +85,13 @@ def test_get_users_by_location_and_role(self):
         CommonTestCases.user_token_assert_in(
             self, get_users_by_location_and_role, "No users found"
             )
+
+    def test_filter_users_by_location_no_users(self):
```
Comment (Bonifase) [inline tests/test_user/test_query_user.py]: Make this test function name more descriptive. The name is different from what is being tested

Comment (bevkololi) [reply tests/test_user/test_query_user.py]: Thanks @Bonifase, all the above have been named appropriately now.

File: tests/test_block/test_create_block.py [hunk 6]
```diff
@@ -104,3 +105,23 @@ def test_delete_non_existent_block(self):
             delete_non_existent_block,
             delete_non_existent_block_response
         )
+
+    def test_delete_block_different_location(self):
```
Comment (Bonifase) [inline tests/test_block/test_create_block.py]: Use more descriptive names for your tests to reflect what is being tested for. E.g `test_delete_block_in_different_location_throws_error`

File: tests/test_room_resource/test_delete_resource.py [hunk 7]
```diff
@@ -32,3 +32,10 @@ def test_non_existant_deleteresource_mutation(self):
             delete_non_existant_resource,
             "Resource not found"
         )
+
+    def test_deleteresource_in_room_that_is_not_in_admin_location(self):
```
Comment (dnuwa) [inline tests/test_room_resource/test_delete_resource.py]: @bevkololi I think you meant to name it `test_delete_resource_in_room_that_is_not_in_admin_location`.
Also checkout line `14` and `35` in `test_update_room_resource.py` for the same naming issue

File: tests/test_block/test_create_block.py (hunk 6 above)
Comment (bevkololi) [reply tests/test_block/test_create_block.py]: @Bonifase, I've updated the names. Some were too long and didn't leave room for adding `throws_errors`.

File: tests/test_room_resource/test_delete_resource.py (hunk 7 above)
Comment (bevkololi) [reply tests/test_room_resource/test_delete_resource.py]: Thanks @dnuwa, I've updated this.

**Pull request merged**